components/Tabs2/tests/unit/MDCTabBarViewTests.m

```diff
@@ -17,12 +17,131 @@
 #import "MDCTabBarView.h"
 
 @interface MDCTabBarViewTests : XCTestCase
+
+@property(nonatomic, strong) MDCTabBarView *tabBarView;
+
+@property(nonatomic, strong) UITabBarItem *itemA;
+
+@property(nonatomic, strong) UITabBarItem *itemB;
+
+@property(nonatomic, strong) UITabBarItem *itemC;
+
 @end
 
 @implementation MDCTabBarViewTests
 
+- (void)setUp {
+  [super setUp];
+
+  _tabBarView = [[MDCTabBarView alloc] init];
+  _itemA = [[UITabBarItem alloc] initWithTitle:@"A" image:nil tag:0];
+  _itemB = [[UITabBarItem alloc] initWithTitle:@"B" image:nil tag:0];
+  _itemC = [[UITabBarItem alloc] initWithTitle:@"C" image:nil tag:0];
+}
+
+- (void)tearDown {
+  _itemA = nil;
```

Comment:
Please use property setters here and throughout.

Reply:
Fix applied.